### DISCUSSION TITLE
Battery discharging is not happened in Andes version 1.7.6

### DISCUSSION THREAD

**Discussion by rana253** [CURENT/andes#374] [Q&A]
Hi Dr. Cui,

Hope you are doing good..
I am working on ANACONDA 3, and installed 1.7.6 in my local PC. Same configuration is also installed on my colleagues PC. We are running the same `use case` and giving battery charging command., i.e.,  **system.ESD1.alter("pref0", "ESD1_1",-0.004**). However, On his local PC, command is working fine and battery is charging .However, on my PC, battery power is constant, when the command is executed.

Could you please suggest the reason for this `maloperation of the command` on my PC?

Regards,
Rana Choudhury

**Comment by cuihantao** [comment]
If the same configuration “$HOME/.andes/andes.rc” is used, I’m not sure about the cause. you might try reinstalling by following the same procedure your colleague did.

Regards,
Hantao Cui

**Reply by rana253** [reply]
Thank you for your prompt response.

My colleagues system contains **andes.rc** in the said path i.e., `“$HOME/.andes/andes.rc” `. However, the **andes.rc** file is not available in my PC. Please suggest to resolve this issue.

Regards,
Rana Choudhury

**Comment by cuihantao** [comment]
You can copy it over.

Regards,
Hantao Cui

**Comment by rana253** [comment]
Thank you for your response. 

Regards,
Rana Choudhury
